src/util/migrations/mariadb/1661785289467-guild-member-profiles.ts

```diff
@@ -0,0 +1,60 @@
+import { MigrationInterface, QueryRunner } from "typeorm";
+
+export class guildMemberProfiles1661785289467 implements MigrationInterface {
+    name = 'guildMemberProfiles1661785289467'
+
+    public async up(queryRunner: QueryRunner): Promise<void> {
+        await queryRunner.query(`
+            ALTER TABLE \`connected_accounts\` DROP COLUMN \`external_id\`
+        `);
+        await queryRunner.query(`
+            ALTER TABLE \`connected_accounts\` DROP COLUMN \`integrations\`
+        `);
```

Comment:
Whats up with the edits to the connected_accounts table here ( and elsewhere )? Is this because your database was configured for the connections PR you made? haha

Reply:
I thought I removed all of these lmao. yeah, I forgot to clean my DB.